bin/start_backend.sh

```diff
@@ -206,14 +215,47 @@ fi
 if [ ${RUN_LOG_CONSOLE} -eq 1 ] ; then
     # force glog output to console (stderr)
     export GLOG_logtostderr=1
-else
-    # redirect stdout/stderr to ${LOG_FILE}
-    exec &>> ${LOG_FILE}
 fi
 
-echo "start time: $(date), server uptime: $(uptime)"
+check_health() {
```

Comment:
This is really tricky. I am inclined not to do so in start script. the `--daemon` is a simplest way to provide a `daemonized` launched solution, but usually it is not enough in production without watchdog. so A supervisor or systemd-service will be prefered to launch the service without `--daemon` and additional healthy/probing methods can be applied out of the start script and control whether need to restart/kill the process with `supervisorctl restart <fe/be/cn>` or `systemctl restart <fe/be/cn>`.